# Fix: src/parser.hh:1407:29: error: no member named 'max' in namespace 'std'

--- a/src/parser.hh
+++ b/src/parser.hh
@@ -18,6 +18,7 @@
 
 #pragma once
 
+#include <algorithm>
 #include <cstdint>
 #include <cstdio>
 #include <optional>
